sdks/python/apache_beam/pipeline_test.py

```diff
@@ -285,6 +299,30 @@ def raise_exception(exn):
   #   p = Pipeline('EagerRunner')
   #   self.assertEqual([1, 4, 9], p | Create([1, 2, 3]) | Map(lambda x: x*x))
 
+  def test_ptransform_overrides(self):
+
+    class MyParDoMatcher(object):
+
+      def __call__(self, applied_ptransform):
```

Comment:
Alternative, you could define this as

```
def my_par_do_matcher(applied_ptransform):
  return isinstance(applied_ptransform.transform, DoubleParDo)
```

and then below you would have

`return my_par_do_matcher`

(The lack of first-class functions is what necessitates a class in Java.)